src/util/contentful/richTextRenderer.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/prefer-default-export */
+/*
+ * Custom renderer for Contentful Rich Text content
+ * Has the ability to return html for embedded assets
+ * and entries inside of that rich text content.
+ * Docs: https://github.com/contentful/rich-text/tree/master/packages/rich-text-html-renderer#usage
+ */
+
+import { BLOCKS, INLINES } from '~/@contentful/rich-text-types';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+/**
+ * Returns html string to render a contentful asset as a vue component
+ *
+ * @param {object} contentfulAssetObject Content of a contentful asset object
+ * @returns {string} String of html to render
+ */
+const assetRenderer = contentfulAssetObject => {
+	const isVideo = contentfulAssetObject?.file?.contentType.includes('video');
+	const isImage = contentfulAssetObject?.file?.contentType.includes('image');
+	if (isImage) {
+		return `
+			<kv-contentful-img
+				contentful-src="${encodeURI(contentfulAssetObject?.file?.url)}"
+				alt="${contentfulAssetObject?.description}"
+				height="${contentfulAssetObject?.file?.details?.image?.height}"
```

Comment:
This will be the original upload width/height of the image?

Reply:
Yes. This is for inline images which should be reasonably sized, but I'm open to suggestions